Microsoft.Toolkit.Uwp.UI.Controls/RemoteDevicePicker/RemoteDevicePicker.cs

```diff
@@ -0,0 +1,374 @@
+        }
+
+        private void RemoteDevicePicker_PrimaryButtonClick(ContentDialog sender, ContentDialogButtonClickEventArgs args)
+        {
+            if (_listDevices.SelectedItems.Count > 0)
+            {
+                _listDevices.SelectedItems?.Clear();
+            }
+        }
+
+        /// <summary>
+        /// Initiate Picker UI
+        /// </summary>
+        /// <returns>A <see cref="Task"/> representing the asynchronous operation.</returns>
+        public async Task<IEnumerable<RemoteSystem>> PickDeviceAsync()
+        {
+            await ShowAsync();
+            IEnumerable<RemoteSystem> devices = ReturnDevices();
+            return devices;
+        }
+
+        internal IEnumerable<RemoteSystem> ReturnDevices()
+        {
+            List<RemoteSystem> returnValue = new List<RemoteSystem>();
+            foreach (RemoteSystem item in _listDevices.SelectedItems)
+            {
+                returnValue.Add(item);
+            }
+
+            return returnValue.AsEnumerable();
+        }
+
+        /// <inheritdoc />
+        protected override void OnApplyTemplate()
+        {
+            UnhookEvents();
+
+            _listDevices = GetTemplateChild("PART_ListDevices") as ListView;
+            _listDeviceTypes = GetTemplateChild("PART_ListDeviceTypes") as ComboBox;
+            _progressRing = GetTemplateChild("PART_ProgressRing") as ProgressRing;
+            _commandSpace = GetTemplateChild("CommandSpace") as Grid;
+            _deviceDiscovery = GetTemplateChild("DiscoveryType") as ComboBox;
+            _deviceStatus = GetTemplateChild("StatusType") as ComboBox;
+            _authorizationType = GetTemplateChild("AuthorizationType") as ComboBox;
+            _advancedFiltersGrid = GetTemplateChild("AdvancedFiltersGrid") as Grid;
+
+            var deviceList = typeof(RemoteSystemKinds).GetProperties().Select(a => a.Name).ToList();
+            deviceList.Add("All");
+            deviceList.Add("Unknown");
+
+            if (_listDeviceTypes != null)
+            {
+                _listDeviceTypes.ItemsSource = deviceList.OrderBy(a => a.ToString());
+                _listDeviceTypes.SelectionChanged += ListDeviceTypes_SelectionChanged;
+            }
+
+            if (_listDevices != null)
+            {
+                _listDevices.SelectionChanged += ListDevices_SelectionChanged;
+                _listDevices.ContainerContentChanging += ListDevices_ContainerContentChanging;
+
+                _listDevices.SelectionMode = SelectionMode == RemoteDeviceSelectionMode.Single ? ListViewSelectionMode.Single : ListViewSelectionMode.Multiple;
+                _listDevices.IsMultiSelectCheckBoxEnabled = SelectionMode == RemoteDeviceSelectionMode.Multiple ? true : false;
+                if (_listDevices.SelectionMode != ListViewSelectionMode.Single)
+                {
+                    _listDevices.DoubleTapped += ListDevices_DoubleTapped;
```

Comment:
You want this in the else so it double-taps in Single Mode, this enables it for Multiple currently.

Reply:
Fixed.